fvm/src/kernel/mod.rs

```diff
@@ -191,11 +191,11 @@ pub trait ActorOps {
     #[cfg(feature = "m2-native")]
     fn install_actor(&mut self, code_cid: Cid) -> Result<()>;
 
-    /// Returns whether the supplied code_cid belongs to a known built-in actor type.
-    fn get_builtin_actor_type(&self, code_cid: &Cid) -> Option<actor::builtin::Type>;
+    /// Returns the actor's "type" (if builitin) or 0 (if not).
```

Comment:
```suggestion
    /// Returns the actor's "type" (if built-in) or 0 (if not).
```

Reply:
Would be useful to link to the enumeration to provide guidance on how to interpret this number.